jena-arq/src/main/java/org/apache/jena/riot/web/HttpNames.java

```diff
@@ -63,21 +63,22 @@
     public static final String paramDefaultGraphURI = "default-graph-uri" ;
     public static final String paramNamedGraphURI   = "named-graph-uri" ;
     public static final String paramTarget          = "target" ;
-    
-    
+
+
     public static final String paramStyleSheet      = "stylesheet" ;
     public static final String paramAccept          = "accept" ;
-    public static final String paramOutput1         = "output" ;        // See Yahoo! developer: http://developer.yahoo.net/common/json.html 
-    public static final String paramOutput2         = "format" ;        // Alternative name 
+    public static final String paramOutput1         = "output" ;        // See Yahoo! developer: http://developer.yahoo.net/common/json.html
```

Comment:
For later maybe; this link is 404. Int he wayback machine several 302 entries. The 302'd page was 404. Last good crawled page for the original link I found is from 2006: http://web.archive.org/web/20060324091551/http://developer.yahoo.com/common/json.html

Reply:
Good find. Reading the link, not sure the comment adds much - "output" is for any syntax nowadays. Removed.